greynoise/info.json

```diff
@@ -0,0 +1,305 @@
+                    "description": "IP address to lookup in GreyNoise."
+                }
+            ]
+        },
+        {
+            "operation": "lookup_ip_community",
+            "title": "Lookup GreyNoise IP Community Information",
+            "description": "Lookup up the provided IP in the GreyNoise Community API endpoint",
+            "enabled": true,
+            "category": "investigation",
+            "annotation": "ip_reputation",
+            "output_schema": {
+                "ip": "",
+                "noise": "",
+                "riot": "",
+                "classification": "",
+                "name": "",
+                "link": "",
+                "last_seen": "",
+                "message": ""
+            },
+            "parameters": [
+                {
+                    "title": "IP Address",
+                    "required": true,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "ip_address",
+                    "value": "",
+                    "description": "IP address to lookup in GreyNoise."
+                }
+            ]
+        },
+        {
+            "operation": "lookup_ip_quick",
+            "title": "Lookup GreyNoise IP Quick Information",
+            "description": "Lookup up the provided IP in the GreyNoise Quick API endpoint",
+            "enabled": true,
+            "category": "investigation",
+            "annotation": "ip_reputation",
+            "output_schema": {
+                "ip": "",
+                "code": "",
+                "code_message": "",
+                "riot": "",
+                "noise": ""
+            },
+            "parameters": [
+                {
+                    "title": "IP Address",
+                    "required": true,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "ip_address",
+                    "value": "",
+                    "description": "IP address to lookup in GreyNoise."
+                }
+            ]
+        },
+        {
+            "operation": "gnql_query",
+            "title": "GreyNoise GNQL Query",
+            "description": "Query the GreyNoise GNQL API endpoint",
+            "enabled": true,
+            "category": "investigation",
+            "annotation": "query",
+            "output_schema": {
+                "complete": "",
+                "scroll": "",
+                "query": "",
+                "count": "",
+                "message": "",
+                "data": ""
+            },
+            "parameters": [
+                {
+                    "title": "Query",
+                    "required": true,
+                    "editable": true,
+                    "visible": true,
+                    "type": "text",
+                    "name": "query",
+                    "value": "",
```

Comment:
could also use the "placeholder" key to show an example the way it is shown in the GreyNoise User Inrerface
 "placeholder": "eg, \"actor\": \"A10 Networks\""